src/Bundle/Doctrine/ORM/Driver.php

```diff
@@ -43,8 +43,11 @@ public function getDataSource(array $configuration, Parameters $parameters): Dat
         /** @var EntityRepository $repository */
         $repository = $manager->getRepository($configuration['class']);
 
+        $fetchJoinCollection = $configuration['pagination']['fetch_join_collection'] ?? true;
+        $useOutputWalkers = $configuration['pagination']['use_output_walkers'] ?? true;
+
         if (!isset($configuration['repository']['method'])) {
-            return new DataSource($repository->createQueryBuilder('o'));
+            return new DataSource($repository->createQueryBuilder('o'), $fetchJoinCollection, $useOutputWalkers);
```

Comment:
Just a minor thought, but perhaps it would be better to pass the whole configuration array into the data source?

Reply:
I have no strong opinion about that change. But now, it looks that every operation on the configuration array occurs in Driver, so it makes sense to do it there and pass only proper parameters to DataSource. Do you see any benefits or potential benefits of changing it?